### PR TITLE
Feature: Blockquote image variants

### PR DESCRIPTION
**Pull request by bspeare**
This pull request updates the blockquote to include variants using a profile image. 

# How to test

- Review the code in "files changed" based on the criteria outlined below in the "how to review" section
- Test out https://uids.brand.uiowa.edu/branches/feature_blockquote_prototyping/components/detail/blockquote--default.html and compare against https://uids.brand.uiowa.edu/components/detail/blockquote--default.html
- Make suggested fixes in your review or pull the code down locally and adjust the code and push up your suggested fixes.
<details>

- https://uids.brand.uiowa.edu/branches/feature_blockquote_prototyping/components/detail/blockquote--left-with-image-aligned-to-the-left-of-the-content.html
- https://uids.brand.uiowa.edu/branches/feature_blockquote_prototyping/components/detail/blockquote--left-with-image-in-footer.html
- https://uids.brand.uiowa.edu/branches/feature_blockquote_prototyping/components/detail/blockquote--center-with-image-above.html
- https://uids.brand.uiowa.edu/branches/feature_blockquote_prototyping/components/detail/blockquote--center-with-image-in-footer.html
- https://uids.brand.uiowa.edu/branches/feature_blockquote_prototyping/components/detail/blockquote--right-with-image-to-the-right-of-the-content.html
- https://uids.brand.uiowa.edu/branches/feature_blockquote_prototyping/components/detail/blockquote--right-with-image-in-footer.html

</details>

# How to review

- **Review the component in the browser:**
  - View the component in more than one browser, preferably Safari, Chrome, and Firefox.  
  - Test each variant in mobile and desktop views and look for any odd formatting.
- **Review `scss` code changes:**   
  - Are BEM modifier and elements used in the right way? https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/
  - Is the component named correctly? Does the name of the component correctly describe the design semantics?
  - Do the class names make sense?  Could they be expanded/shortened?  Are there any namespace collisions with other components?
  - Do there need to be more/less comments?
  - Check the [variables.scss](https://github.com/uiowa/uids/blob/main/src/assets/scss/_variables.scss) and [utilities.scss](https://github.com/uiowa/uids/blob/main/src/assets/scss/_utilities.scss) files to reuse common variables and mixins
  - Is there any redundant code? Is there code that could be abstracted out into a common reusable class?
  - Use [logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties) like `text-align: start` instead of `text-align: left`
  - Review how px, em, rem, %, vh, vw are used.   
    - Use `rem` for font size, padding, and margin and have a few defined in `[variables.scss](https://github.com/uiowa/uids/blob/main/src/assets/scss/_variables.scss)`
    - Use `%` when specifying grid columns, flex layouts, and anything else necessary to define a size as relative to an element's parent object
    - Use `em` mostly when defining the browser breakpoint size
    - Use `px` for values like `border`
    - Use `vw` on an item that needs to be styled relative to the viewport’s width
    - Use `vh` on an item that needs to be styled relative to the viewport’s height
  - And finally, could the Sass formatting be improved? Look for examples at https://github.com/airbnb/css for possible formatting suggestions.
- **Review `yml` changes:**
  - Are all the possible variants represented?
  - Is there any redundant code? 
  - Can the variant description be improved?

### CODE REVIEW

File: src/components/blockquote/blockquote.scss [hunk 1]
```diff
@@ -62,23 +70,171 @@ blockquote {
...
     border-left: none;
-    border-right: 8px solid $primary;
+    border-right: 10px solid $primary;
     @include padding($top: $xsm, $bottom: $xsm, $left: 0, $right: $lg);
   }
 }
+
+// Shared styles for above and below options.
+blockquote.blockquote--img-above,
+blockquote.blockquote--img-below {
```
Comment (pyrello) [inline src/components/blockquote/blockquote.scss:95]: Is there a reason not to include this inside the previous `blockquote` section?

Comment (bspeare) [reply src/components/blockquote/blockquote.scss:95]: Good question/observation, I'll see if I can refactor that!

Comment (bspeare) [reply src/components/blockquote/blockquote.scss]: This should be resolved if you have time to take another look.

**Comment by GaryRidgway** [comment]
I have walked through all of the bullet points, asked questions where needed, and found that this PR seems pretty airtight to me. Looks great Ben!

**Comment by dnkemper** [comment]
with config split 2 being merged, @joewhitsitt said to wait to merge since will need to possibly remove site split perms

**Comment by dnkemper** [comment]
then again, I'm sure this doesn't matter regarding that, right?

**Comment by dnkemper** [comment]
JK. Def thought this was other PR haha

**Review by GaryRidgway — APPROVED** [review]
Reviewed and approved!

**Pull request merged**